docs/code/README.md

```diff
@@ -0,0 +1,23 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Code
+path_base_for_github_subdir:
+  from: tmp/semconv/docs/code/_index.md
+  to: code/README.md
+--->
+
+# Semantic Conventions for Code
+
+**Status**: [Experimental][DocumentStatus]
+
+This document defines semantic conventions for source code.
+
+> **Warning**
+>
+> Existing code instrumentations that are using
+> [v1.29.0 of code conventions](https://github.com/open-telemetry/semantic-conventions/blob/v1.29.0/docs/attributes-registry/code.md)
```

Comment:
maybe 

```suggestion
> [v1.29.0 of `code` conventions](https://github.com/open-telemetry/semantic-conventions/blob/v1.29.0/docs/attributes-registry/code.md)
```

? 

We don't do it in other warning, but "code conventions" seems to be too generic.